mongodb_depot.go

```
@@ -124,6 +124,35 @@ func (m *mongoDepot) Check(tag *depot.Tag) bool {
 	}
 }
 
+// CheckWithError returns whether the user and data specified by the tag exists
+// as well as an error in the case of an internal error.
+func (m *mongoDepot) CheckWithError(tag *depot.Tag) (bool, error) {
+	name, key, err := getNameAndKey(tag)
+	if err != nil {
+		return false, nil
```

Same question as above about ignoring the error.